components/main-chef-wrapper/main.go

```diff
@@ -47,6 +49,21 @@ func main() {
 	case "report", "capture":
 		cmd = exec.Command(dist.AnalyzeExec, allArgs...)
 
+	case "push", "push-archive":
+		cmd = exec.Command(dist.WorkstationExec, allArgs...)
+		runCmd(cmd)
+		if validateRolloutSetup() {
+			policyGroup := fmt.Sprintf("-g%s", allArgs[1])
+			policyLockFile := fmt.Sprintf("-l%s", allArgs[2])
```

Comment:
formatted strings should probably be quoted in output, in case of something like a filename with spaces.

Reply:
Used a different approach in the updates now.